packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/guides/Migration.tool.md

```diff
@@ -0,0 +1,519 @@
+# Terra Functional Testing
+
+We're excited to announce the release of @cerner/terra-functional-testing which will be replacing the automation testing responsibilities of terra-toolkit. We've made some changes and gave our testing infrastructure a much needed upgrade. This document will outline the major changes and provide useful tips for upgrading.
+
+## Introduction
+
+Since terra-toolkit was originally released in 2017 many new features have been added to create an all-in-one development toolkit. These new features introduced responsibilities for the development site, webpack, jest, reporting, and automation testing. Bundling all these responsibilities into single package has made it difficult to introduce new changes. With the upgrade to WebDriverIO v6 we decided to use this as an opportunity for restructuring our development tools.
+
+## New Package Structure
+
+The [terra-toolkit](https://github.com/cerner/terra-toolkit) repository has been converted into a [lerna monorepo](https://github.com/lerna/lerna) that contains independent packages for each of our development tools. The [terra-toolkit development package](https://www.npmjs.com/package/terra-toolkit) has been moved into the [terra-toolkit-boneyard](https://github.com/cerner/terra-toolkit-boneyard) repository and will receive minor updates and bug fixes as teams begin the transition to terra-functional-testing.
+
+Moving forward, development responsibilities have been separated out into individual packages that encapsulate specific functionality. As we transition packages into the terra-toolkit monorepo each package will be scoped under the [@cerner](https://www.npmjs.com/org/cerner) NPM organization. For example, automation testing has been moved into @cerner/terra-functional-testing and webpack concerns have been moved into [@cerner/webpack-config-terra](/dev_tools/cerner/webpack-config-terra/about).
```

Comment:
Is `/dev_tools/cerner/webpack-config-terra/about` meant to be a relative path? I see it uses the full path (https://engineering.cerner.com/terra-ui/dev_tools/cerner/webpack-config-terra/about) in one of the other references below.

Reply:
It's a relative path. I'll launch the site locally and double check each of these routes if I can get it running